docs/build-the-docs.md

```diff
@@ -1,16 +1,30 @@
 # Building the Documentation
 
-You're now ready to build (generate) the ownCloud documentation.
+## Install Antora's Dependencies
+
+Before you can build the documentation, you need to install the required dependencies.
+To install them, you need to run `yarn install`.
+This will install all the dependencies specified in `package.json`.
+**Note:** If your environment already has GNU make installed, you can run `make install` instead.
+
+```console
+make install
+```
+
+With the dependencies installed, you are now ready to build (generate) the ownCloud documentation.
+
+## Generating the Documentation
+
 The documentation can be generated in HTML and PDF formats.
 
-## Generating HTML Documentation
+### Generating HTML Documentation
 
 There are two ways to generate the documentation in HTML format:
 
 - Using ownCloud's custom Antora Docker Container
 - Running Antora from the Command-Line
 
-### Using the Docker Container
+#### Using the Docker Container
 
 To build the documentation using the Docker container, from the command line, in the root of the docs directory, run the following command:
 
```

Comment:
below in line 33/34 you find environment variables set for `lunr`. Is this still valid?

Reply:
Forget my comment, that is fixed in #475